pages/docs/couple-your-code/couple-your-code-gradient-data.md

```diff
@@ -81,20 +81,17 @@ For the example, you can use the following `precice-config.xml` (note the versio
 ```xml
 <?xml version="1.0"?>
 
-<precice-configuration>
-
-  <solver-interface dimensions="3" experimental="on">
-
+<precice-configuration experimental="on">
```

Comment:
Further below in the waveform section, it is `experimental="true"`. Lets be consistent with the value of the tag. Also, is coupling gradient data still experimental?

Reply:
This PR is only moving things around, but good catch anyway! :D Fixed.